### PR TITLE
Dicoba lagi

### PR DESCRIPTION
**Pull request by AzharyusufA**
Saya baru tau rasanya jd minoritas.

### CODE REVIEW

File: Adri.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Apa aja");
```
Comment (rochafialvin) [inline Adri.js:1]: Revisi. Minimal 3 kata

**Review by ridwanrmd — APPROVED** [review]
permen permen apa yang besar..



candy borobudur

**Review by frdzihza — APPROVED** [review]

**Review by Bismvrckz — APPROVED** [review]

**Review by Revirdi — APPROVED** [review]

**Comment by mohrezamau** [comment]
I Sexually Identify as an Attack Helicopter

WEE WOO WEE WOO

ALERT! COMEDY GOD HAS ENTERED THE BUILDING! GET TO COVER!

steps on stage

Bystander: "Oh god! Don't do it! I have a family!"

Comedy God: "Heh..."

adjusts fedora

the building is filled with fear and anticipation

God and Jesus himself looks on in suspense

comedy god clears throat

everything is completely quiet not a single sound is heard

world leaders look and wait with dread

everything in the world stops

nothing is happening

comedy god smirks

no one is prepared for what is going to happen

comedy god musters all of this power

he bellows out to the world

"ATTACK"

absolute suspense

everyone is filled with overwhelming dread

"HELICOPTER"

all at once, absolute pandemonium commences

all nuclear powers launch their nukes at once

giant brawls start

43 wars are declared simultaneously

a shockwave travels around the earth

earth is driven into chaos

humanity is regressed back to the stone age

the pure funny of that joke destroyed civilization itself

all the while people are laughing harder than they ever did

people who aren't killed die from laughter

literally the funniest joke in the world

then the comedy god himself posts his creation to reddit and gets karma

**Review by Bismvrckz — APPROVED** [review]
FBI Open up

**Pull request merged**